cfgov/v1/util/migrations.py

```diff
@@ -57,3 +61,33 @@ def get_or_create_page(apps, page_cls_app, page_cls_name, title, slug,
     parent_page.save()
 
     return page
+
+
+def migrate_stream_field(page, field_name, field_type, mapper):
+    """ Migrate a stream field of the name and type belonging to
+        the page using the mapper function """
+    old_stream_data = get_page_stream_data(page, field_name)
+    new_stream_data = []
+
+    migrated = False
+    for field in old_stream_data:
+        if field_type == field['type']:
+            field['value'] = mapper(page, field['value'])
+            migrated = True
+
+        new_stream_data.append(field)
+
+    if migrated:
+        print('migrated page {}'.format(page.slug))
+        set_page_stream_data(page, field_name, new_stream_data)
+
+
+@transaction.atomic
+def migrate_page_types_and_fields(apps, page_types_and_fields, mapper):
+    """ Migrate the fields of a wagtail page type using the given mapper
+        function. page_types_and_fields should be a list of 3-tuples
+        providing ('PageType', 'field_name', 'field type'). """
+    for page_type, field_name, field_type in page_types_and_fields:
+        page_cls = imported_apps.get_model('v1', page_type)
```

Comment:
Primarily because I just adapted the code in [0011_hero_refactor_data.py](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/migrations/0011_hero_refactor_data.py).

I'll modify it to use `apps`.